rbac/management/tenant_service/v2.py

```diff
@@ -71,6 +71,31 @@ def bootstrap_tenant(self, tenant: Tenant, force: bool = False) -> BootstrappedT
         except TenantMapping.DoesNotExist:
             return self._bootstrap_tenant(tenant)
 
+    def create_ungrouped_workspace(self, org_id) -> Workspace:
+        """Util for creating ungrouped workspace. Can be removed once ungrouped workspace has gone."""
+        tenant = Tenant.objects.get(org_id=org_id)
+        default = Workspace.objects.get(tenant=tenant, type=Workspace.Types.DEFAULT)
+        ungrouped_hosts = Workspace.objects.create(
```

Comment:
do we allow only one `Workspace.Types.UNGROUPED_HOSTS` for particular org ? if so do we need cover it somehow here ?